SEP-0002-revised.md

```diff
@@ -0,0 +1,253 @@
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project exists for the benefit of the whole community: users, developers, and members of the wider solar physics and scientific software community.
+This community is the ultimate authority in how the project is run, this authority is exercised through the processes outlined in this document, primarily to the Steering Committee and the Project Members.
+Whenever possible all decisions taken should be done by consensus, considering input from both active contributors and the wider community, rather than taken in isolation by the projects governance.
+
+## Advisory Board
+The primary functions of the advisory board are: to advise the steering committee on management of the project, find funding for the project to ensure the core aims of the project can be met, nominate a Ombudsperson, and to ensure that the governance of the project is running as intended.
+The board is self-electing, with members appointed by a vote of the existing board.
+
+* **Membership Limit**: The board shall consist of at least 3 members and no more than 7 members.
+The board may increase or decrease the number of people serving on the board.
+
+* **Term Length**: All members shall serve a maximum of a two-year term (from their appointment), however the term may be extended until a successor has been appointed.
+A member may be re-appointed at the end of their term.
+
+* **Staggering**: Members terms shall be staggered so that approximately half the number of members will end their terms in any given year.
+
+* **Vacancies**: The board may fill vacancies or may appoint new members to fill a previously unfilled board position, subject to the maximum number of members.
+The board shall fill vacancies as soon as possible.
+The board should work to ensure that the composition of the board represents a diverse cross-section of the scientific community with an interest in SunPy.
+This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+
+* **Removal**: A board member may be removed by a two-thirds majority vote of the full board, if before any meeting of the board at which a vote on removal will be made the member in question is given electronic or written notification of the board’s intention to discuss their case and is given the opportunity to be heard at a meeting of the board.
```

Comment:
This is an awfully written sentence, and needs to be broken up into two or three sentences.